scripts/jest/TestFlags.js

```diff
@@ -50,9 +50,6 @@ const environmentFlags = {
   FIXME: false,
   TODO: false,
 
-  // Turn these flags back on (or delete) once the effect list is removed in
-  // favor of a depth-first traversal using `subtreeTags`.
-  dfsEffectsRefactor: true,
```

Comment:
Also should be able to remove this, which is only used for strict effects tests.